runscripts/nemo/nemo4-GYRE_PISCES.yaml

```diff
@@ -0,0 +1,82 @@
+# runscript to run the GYRE_PISCES demo case that comes with NEMO4.2
```

Comment:
If this is a runscript that could (at some point in my dreamy-future) be included in CI, please let me know.

Reply:
Not at the moment, as we are still in the "alpha" stage with NEMO4.2.x